src/components/Search/SearchListWithHeader.tsx

```diff
@@ -166,7 +191,16 @@ function SearchListWithHeader(
                 onSelectAll={toggleAllTransactions}
                 isMobileSelectionModeActive={isMobileSelectionModeActive}
             />
-
+            <ConfirmModal
+                isVisible={deleteExpensesConfirmModalVisible}
+                onConfirm={handleDeleteExpenses}
+                onCancel={handleOnCancelConfirmModal}
```

Comment:
FYI: OnModalHide is not handled here causing this issue - https://github.com/Expensify/App/issues/45447